azure/requests.yaml

```diff
@@ -0,0 +1,6 @@
+releases:
+    - name: "> 11.4.0, < 12.0.0"
```

Comment:
I am not sure how much we already do with azure CNI here. I don't think calico does any IPAM calculations here.